src/menu/ide_menu.rs

```diff
@@ -807,6 +811,24 @@ impl Menu for IdeMenu {
 
             self.working_details.space_left = space_left;
             self.working_details.space_right = space_right;
+
+            let available_lines = painter
+                .remaining_lines()
+                .min(self.default_details.max_completion_height)
+                .saturating_sub(1);
```

Comment:
for some reason this is always 1 higher than `available_lines` from the `menu_string` function, not sure why that is the case as both should be the lines (not including the prompt).

Thats why im subtracting 1 here.

Reply:
It seems like the prompt is indeed being included in `painter.remaining_lines()`. If I resize my terminal so that the prompt takes up 2 or 3 lines, `painter.remaining_lines()` seems to be 2 or 3 more lines too big.

Reply:
Here's a recording demonstrating that: https://asciinema.org/a/j8bRJs2t2RrIvUGlIFrXNav20. Notice how the suggestion `foo9` isn't visible when selected

Reply:
ah ok i see

looks like``Painter::remaining_lines`` does not consider the actual prompt (only the location of the prompt, e.g if its not at the top of the terminal)

Reply:
should be fixed with the latest commit but ill do some more testing tomorrow as i did some changes in the Painter